packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/activeDirectories/add/federation-add.controller.js

```diff
@@ -43,6 +45,7 @@ export default class FederationAddCtrl {
   }
 
   onSubmit() {
+    this.trackClick('add-active-directory::confirm');
```

Comment:
If we are using at multiple places , may be we can create prefix and use it.
```suggestion
    const prefix = 'add-active-directory'
    this.trackClick('add-active-directory::confirm');
    this.trackClick(`${prefix}::confirm`);
```

Reply:
It's applied.